Localization/de.lproj/Localizable.strings

```diff
@@ -40,30 +42,30 @@
 "setup.welcome.createButton" = "Erstellen Sie eine neue Wallet";
 "setup.welcome.createDescription" = "Erstellen Sie eine neue Wallet auf Ihrem iOS-Gerät.";
 "setup.welcome.restoreButton" = "Stellen Sie Ihre Wallet wieder her";
-"setup.welcome.restoreDescription" = "iOS-wallet verloren? Stellen Sie es hier wieder her.";
+"setup.welcome.restoreDescription" = "Haben Sie Ihre iOS-Wallet verloren? Stellen Sie Ihre Wallet hier wieder her.";
```

Comment:
This text is waaaayyy to long

Reply:
We had the same issues with Japanese I fixed before the commit. 
What TO DO: I could bring only a new keys translations but I couldn't fix issues like your example, cause I don't know the context in those languages.